frontend/src/scenes/insights/utils/cleanFilters.test.ts

```diff
@@ -68,6 +69,25 @@ describe('cleanFilters', () => {
         expect(cleanedFilters).toHaveProperty('breakdown_type', 'event')
     })
 
+    const breakdownIndexTestCases = [
+        { provided: 0, expected: 0 },
+        { provided: 27, expected: 27 },
+        { provided: null, expected: undefined },
+        { provided: undefined, expected: undefined },
+        { provided: 0, breakdown_type: 'event', expected: undefined },
+    ]
+    breakdownIndexTestCases.forEach((testCase) => {
```

Comment:
Not a fan of this pattern (too much meta-programming for tests) but better than nothing.

Reply:
I _love_ parameterised tests :)

Reply:
I like parameterized tests as long as they're simple - in this case you need to parse 3 layers (this const, the testCase modifications, the test code) to see what the test is doing. It could be a lot simpler! :)